### PR TITLE
Add tests for cache to increase test coverage

### PR DESCRIPTION
**Pull request by rocktavious**

### CODE REVIEW

File: testdata/templates/tiers.tpl [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  "alias": "{{ template "alias1" }}",
+  "description": "Mission-critical service or repository. Failure could result in significant impact to revenue or reputation.",
+  "id": "{{ template "id1" }}",
+  "index": 1,
+  "name": "Tier 1"
+}
+{{ end }}
+{{- define "tier_2" }}
+{
+  "alias": "{{ template "alias2" }}",
```
Comment (matthewbrahms) [inline testdata/templates/tiers.tpl:12]: Nice, templating in templating...template-ception!

**Review by matthewbrahms — APPROVED** [review]
looks good to me

File: level_test.go [hunk 2]
```diff
@@ -114,7 +114,7 @@ func TestUpdateRubricLevel(t *testing.T) {
 	}`
 	response := `{"data": {
 	"levelUpdate": {
-		"level": {{ template "level.tpl" }},
+		"level": {{ template "level_1" }},
```
Comment (eochoa260) [inline level_test.go:117]: These templates are working out nicely! 🎉

**Review by eochoa260 — APPROVED** [review]
LGTM

**Pull request merged**